cmd/gcp-controller-manager/app/ca_cache_test.go

```diff
@@ -25,16 +24,41 @@ import (
 	"path"
 	"testing"
 	"time"
+
+	"k8s.io/cloud-provider-gcp/pkg/nodeidentity"
 )
 
 func TestCACacheVerify(t *testing.T) {
-	var ca fakeCA
+	ca, c, cleanup := initFakeCACache(t)
+	defer cleanup()
+
+	t.Run("valid", func(t *testing.T) {
+		t.Parallel()
+		if err := c.verify(ca.validCert); err != nil {
+			t.Errorf("verifying valid certificate: got %v, want nil", err)
+		}
+	})
+	if err := c.verify(ca.validCert); err != nil {
```

Comment:
Isn't this the same test case as the one in t.Run("valid")?

Reply:
Oops, removed.
This actually surfaced another bug: parallel subtests could run after `defer cleanup()`, which made negative tests fail for the wrong reason.